tensorhive/models/Restriction.py

```diff
@@ -54,6 +59,34 @@ def check_assertions(self):
             assert self.ends_at > datetime.datetime.utcnow(), 'You are trying to edit restriction that has already' \
                                                               ' expired - please do not do that!'
 
+    @hybrid_property
+    def starts_at(self):
+        return self._starts_at
+
+    @hybrid_property
+    def ends_at(self):
+        return self._ends_at
+
+    @hybrid_property
+    def created_at(self):
+        return self._created_at
+
+    @starts_at.setter
+    def starts_at(self, value: str):
+        self._starts_at = DateUtils.try_parse_string(value)
+        if not self._starts_at:
+            log.error('Unsupported type (starts_at={})'.format(value))
+
+    @ends_at.setter
+    def ends_at(self, value: str):
+        self._ends_at = DateUtils.try_parse_string(value)
+
+    @created_at.setter
+    def created_at(self, value: str):
+        self._created_at = DateUtils.try_parse_string(value)
+        if not self._created_at:
```

Comment:
Do we expect any falsy value other than `None` here? If not we could replace this with the following code, so that it explicitly shows the intent of this check.

```suggestion
        if self._starts_at is not None:
```

Reply:
Fixed